_maps/map_files/PubbyStation/PubbyStation.dmm

```diff
@@ -2,19 +2,6 @@
 "aaa" = (
```

Comment:
Nothing was added here?

Reply:
"aaa" = (
/turf/open/space/basic,
/area/space)

What do you mean? There's a space tile here, I didn't add it.

Reply:
I'm using DM to edit the map files, not a text editor. I don't know what "aaa"  means.

Reply:
Do I have to mention your username like this @ShizCalev so you get notified about a comment?

Reply:
I'm saying that you didn't actually change anything on Pubby. lol

Reply:
I'm trying to fix it, map merge keeps deleting lines.

Reply:
I don't want to change anything on Pubby.